api-errors.json

```diff
@@ -195,6 +195,10 @@
         "short_message": "Invalid Barcode",
         "long_message": "The barcode could not be created due to invalid barcode type"
     },
+    "40099": {
```

Comment:
Use code 40053 and move it up. It was stupid to add the code 40098 a few years back. It incorrectly looks like we're almost out of numbers. Perhaps you could also leave a comment if you think that another person will do the same later on.

Reply:
Actually I think that 40049 to 40051 has not been used either 🤦 .